src/components/TokenSelect.js

```diff
@@ -149,7 +77,7 @@ const TokenSelect = (props) => {
 
   // Can't use SafeAreaView because the list view must go until the end of the screen
```

Comment:
💬 What about this comment? Does it still holds true?

Reply:
Not anymore, since we're now using a dedicated lib for this: `react-native-safe-area-context`.
Below a screenshot indicating the results are as expected
![Screenshot_1687964533](https://github.com/HathorNetwork/hathor-wallet-mobile/assets/1299409/7b0e9cc7-cfe6-4699-b814-a5802bd88da3)

Thanks for pointing out! Cleared the obsolete comment